src/components/DisciplineItem.tsx

```diff
@@ -43,7 +43,7 @@ export interface IMenuItemProps {
   onPress: () => void
```

Comment:
Also this interface (line 39) should be renamed. Please name it `DisciplineItemProps`